### PR TITLE
[KYUUBI #2034] Hive Backend Engine - GetPrimaryKeys

### PR DESCRIPTION
**Pull request by KenjiFujima**
### _Why are the changes needed?_

Hive Backend Engine - GetPrimaryKeys.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/GetPrimaryKeys.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import org.apache.kyuubi.operation.OperationType
+import org.apache.kyuubi.session.Session
+
+class GetPrimaryKeys(
+    session: Session,
+    catalogName: String,
+    schemaName: String,
+    tableName: String)
+  extends HiveOperation(OperationType.GET_FUNCTIONS, session) {
```
Comment (pan3793) [inline externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/GetPrimaryKeys.scala:30]: The OperationType should be `UNKNOWN_OPERATION`

**Review by yaooqinn — APPROVED** [review]
LGTM

**Comment by yaooqinn** [comment]
the test failures are relevant please check

File: externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/GetPrimaryKeys.scala (hunk 1 above)
Comment (KenjiFujima) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/GetPrimaryKeys.scala:30]: @pan3793, as offline discussion with @yaooqinn , the type of this operation should be `GET_FUNCTIONS`.

**Comment by zhaomin1423** [comment]
Should we use new tickets to update flink and trino?

**Comment by KenjiFujima** [comment]
@zhaomin1423, if you have interest to support `GetPrimaryKeys` operation in other engines, you could create new tickets for this. Thanks.

**Review by yaooqinn — APPROVED** [review]
lgtm

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**